File: app/src/main/java/com/example/match3game/MainActivity.java
package com.example.match3game;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

/**
 * 主界面 Activity
 * 显示游戏入口和设置
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // 设置点击事件
        Button btnPlay = findViewById(R.id.btn_play);
        btnPlay.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GameActivity.class);
                startActivity(intent);
            }
        });

        Button btnSettings = findViewById(R.id.btn_settings);
        btnSettings.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // TODO: 打开设置界面
            }
        });
    }
}

File: app/src/main/java/com/example/match3game/MainActivity.kt
package com.example.match3game

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

/**
 * 主界面 Activity
 * 显示游戏入口和设置
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // 设置点击事件
        findViewById<View>(R.id.btn_play).setOnClickListener {
            startActivity(Intent(this, GameActivity::class.java))
        }

        findViewById<View>(R.id.btn_settings).setOnClickListener {
            // TODO: 打开设置界面
        }
    }
}
